src/platform/shared/common.c

```diff
@@ -11,6 +11,9 @@ ALIGNED(256) uint8_t OAM[OAM_SIZE] = { 0 };
 ALIGNED(256) uint8_t FLASH_BASE[FLASH_ROM_SIZE_1M * SECTORS_PER_BANK] = { 0 };
 
 u16 INTR_CHECK = 0;
+
+void IntrMain(void);
```

Comment:
In theory should be extern? I know that's implied with functions anyway

Reply:
I mean, in a header file, sure.
But here it's just there for the assignment below that line, but it's in the same file, so it being `extern` would not be accurate.

Reply:
Ohh perhaps I misunderstood, all good!